### PR TITLE
Adds Vox Anesthetics

### PR DESCRIPTION
**Pull request by AffectedArc07**
Does what it says in the title. Adds a set of anesthetics with N2 instead of O2 as the secondary gas.

:cl:
rscadd: Adds vox anesthetics, Two cannisters per anesthetics locker.
/:cl:

### CODE REVIEW

**Comment by Taaketa** [comment]
Aw now my job as atmos is completely gone. I used to vox sleepy gas for medical all the time.

**Comment by Fox-McCloud** [comment]
I detect web editor heresy.

Also, this will still poison them. It'll have oxygen in it, since it calls `..()`

**Comment by FlattestGuitar** [comment]
This still guarantees Vox will take a few points of toxin damage, I don't see the point in using it over ether, if you time everything right.

**Comment by AndriiYukhymchak** [comment]
I use hydrocodone for vox surgeries

**Comment by pinatacolada** [comment]
i really don't like this, are vox so special they deserve an anaesthetic tank just for them? there are so many other ways to anaesthetize a vox for surgery, this isn't needed at all. it just takes away from the uniqueness of dealing with an alien race

**Comment by SamHPurp** [comment]
@Fox-McCloud technically not, they could attach it to the suit slot, while having their currently active internals in their hand, then activate the internals off the anesthetic. 

The problem with anesthesia in general is that there is no penalty for not using them in some way (including Ether).

**Comment by davidchan13** [comment]
@SamHPurp how do you propose getting their anesthetic tank off and their N2 tank on without them taking toxins?

It's not a bad idea, but until someone figures out a way to use internals not on your person (on the floor, another mobs hands, ect...) this isn't ever going to work as intended as there is no way to get the N2 on fast enough before the vox breathes in o2 unless you take them into space or set up an N2 only operating theatre.

**Comment by SamHPurp** [comment]
@davidchan13 Having them do it themselves, they can swap internals without disabling one.

**Comment by deep-fried-faceplant** [comment]
Giving anesthetics to vox would make painkillers and knockout drugs like ether, morphine, and hydrocodone even more obsolete.

**Comment by Doukhan** [comment]
Vox were meant to be this uncommon species of weirdo space raptors now it seems they'l be getting their own bathroom.

**Comment by MischievousGremlin** [comment]
I'd suggest 40u morphine patches put in medical to avoid the addiction thing, ether takes far too long to work.

File: code/game/objects/items/weapons/tanks/tank_types.dm [hunk 1]
```diff
@@ -56,6 +56,21 @@
...
 	air_contents.trace_gases += trace_gas
 
+/obj/item/weapon/tank/anesthetic/vox
+	name = "vox anesthetic tank"
+	desc = "A tank with an N2O/N2 gas mix."
+	icon_state = "oxygen_fr"
+	item_state = "an_tank"
+
+/obj/item/weapon/tank/anesthetic/vox/New()
+	..()
```
Comment (Crazylemon64) [inline code/game/objects/items/weapons/tanks/tank_types.dm]: There's still going to be oxygen in there, so vox will still be poisoned.

**Comment by Crazylemon64** [comment]
Ehhh, this kinda feels like it's "flattening" the differences/inconveniences of being a vox to fit in with more conventional species.

**Comment by SamHPurp** [comment]
@Crazylemon64 its about sanitising the race and making it more humanlike.... which I'm very much against.

**Comment by Fox-McCloud** [comment]
This doesn't look like it's going to fly. Closing.

**Pull request closed without merging**